data.sql

```diff
@@ -2,9 +2,33 @@
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Agumon', '2020-02-03',0,true, 10.23);
+
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Gabumon', '2018-11-15', 2,true, 8);
+
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Pikachu', '2021-01-07', 1,false, 15.04);
+
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Devimon', '2017-05-12', 5,true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Charmander', '2020-02-08', 0,false, -11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Plantmon', '2021-11-15', 2,true, -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Squirtle', '1992-04-02', 3,false, -12.3);
```

Comment:
- [ ] The birth year for this animal should be `1993` instead of `1992`
- [ ] Secondly, the weight should be `-12.13`

Reply:
> * [x]  The birth year for this animal should be `1993` instead of `1992`
> * [x]  Secondly, the weight should be `-12.13`

- [ ] As the previous reviewer suggested, Please work on it. :+1: